### DISCUSSION TITLE
Constant errors when using the plugin in LazyVim

### DISCUSSION THREAD

**Discussion by osenvosem** [mrcjkb/rustaceanvim#454] [Q&A] [answered]
Huge number of these errors while editing and moving through code. Is there any idea how to deal with it?

<img width="1402" alt="350112753-2ea2a74b-9be3-4a47-a8c4-7c978d782b65" src="https://github.com/user-attachments/assets/b6ff6d50-9dc3-4af5-9463-b9a76422faff">

**Comment by mrcjkb** [comment, marked as answer]
Hey 👋 

I don't have the capacity to support all the Neovim distributions out there, so I can only suggest the following:

- Follow the troubleshooting guide starting from a minimal config and plain neovim, as recommended in the bug report form.
- Open a Q&A discussion in the LazyVim repo. People who use LazyVim will be more likely to be able to help you than I am 😀

**Reply by osenvosem** [reply]
Thank you for reply. I posted a question in the LazyVim repo a couple of days ago, but it remains unanswered. That's why I'm here)

**Reply by mrcjkb** [reply]
The error message looks like it comes from a rust-analyzer debug build. 

See also: https://github.com/rust-lang/rust-analyzer/issues/3959
